src/EbmlContexts.cpp

```diff
@@ -45,9 +45,9 @@ static const EbmlSemantic EbmlGlobal_ContextList[2] =
   EbmlSemantic(false, false, EBML_INFO(EbmlVoid)),    ///< EbmlVoid
 };
 
-const EbmlSemanticContext Context_EbmlGlobal = EbmlSemanticContext(0, nullptr, nullptr, *GetEbmlGlobal_Context, nullptr);
+const EbmlSemanticContext Context_EbmlGlobal = EbmlSemanticContext(0, nullptr, nullptr, GetEbmlGlobal_Context, nullptr);
```

Comment:
I do not understand how this can work. The `EbmlSemanticContext` constructor is not changed in this commit. Also other pieces of code might be use the same constructor to create their own context (IIRC we have one in VLC). Will they have to update the code to remove the `*` ? (and have different code depending on the libebml version in use)